.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+# Updating CircleCI configuration from v1 to v2
+version: 2
+jobs:
+  build:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: ls
+
+  test:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          command: docker build -t jupyterhub/jupyterhub .
+      - run:
+          command: docker build -t jupyterhub/jupyterhub-onbuild:${CIRCLE_TAG:-latest} onbuild
+
+  deploy:
+    machine:
+      enabled: true
+    steps:
+      - checkout
```

Comment:
deploy and release shouldn't need checkout steps, only test

Reply:
sounds good, I've removed the checkout step from deploy and release